### ISSUE TITLE
Use KafkaConsumer 0.9+ for Simple Consumer API

### ISSUE THREAD

**Issue by GRpro** [confluentinc/kafka-rest#233]

**Comment by GRpro** [comment]
Opened a PR: https://github.com/confluentinc/kafka-rest/pull/234

**Comment by jeffwidman** [comment]
Is this still relevant now that the 3.2 release switched to supporting the kafka 0.9 broker-based consumer groups?

**Comment by ewencp** [comment]
@jeffwidman Yeah, this is still relevant. The v2 API didn't get [this API](http://docs.confluent.io/current/kafka-rest/docs/api.html#get--topics-(string-topic_name)-partitions-(int-partition_id)-messages?offset=(int)[&count=(int)]) yet, so it is still only supported in v1. It is backed by `SimpleConsumerManager` which still uses the old client.

**Comment by jeffwidman** [comment]
@ewencp so if I understand correctly, if we just want to consume from a topic, we can now use the V2 API. 

But if we want to consume from a specific partition within the topic, we still need to use the V1 API, as the V2 API doesn't support that yet. And, since the V1 API uses the old client, we still would need a zookeeper dependency... 

Is this understanding correct?
